appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
 build: off
 
 test_script:
-  - npm test
```

Comment:
Can we just use GitHub Actions to test Windows? Then we could get rid of this file altogether.